SmithEtAl2024_CSEEnT.tex

```diff
@@ -284,20 +285,125 @@ \subsection{Timeline Comparison}
 
 \subsection{Measuring Fairness}
 
-New fairness metric.
+In order to measure the fairness of work distribution among teammates,
+we sought to compute a metric where values range from 0 to 1, and where values
+in between contain meaningful information.
+Thus, we devised the following \textit{unfairness} metric where $C$:
 
 $$
-{ \sum\limits_{c, x \in C} (c > x \Rightarrow  {c-x})} / {((\left|C\right| -
-1) \cdot \sum\limits_{c \in C} c)}
+\text{unfairness}(C) = \frac{ \sum\limits_{c, x \in C, c > x} (c-x)}{(\left|C\right| -
```

Comment:
I like that!  It really makes it clear (at least to me).  :smile:  We've had a nice example of a "faking" a rationale design process.  I don't think either of us would have written down this nice elegant equation in the first place, but the back and forth has led to something nice.  We can just pretend that we saw it this way all along.  :smile: